eadf/importers.py

```diff
@@ -186,3 +189,53 @@ def fromAngleListData(
     )
 
     return EADF(arrInter, arrAzi, arrCoEle, arrFreqData, arrPos)
+
+def loadData(
```

Comment:
The function name is completely void of information. It should be changed to something more meaningful.